tests/schema_simple.py

```diff
@@ -4,6 +4,10 @@
 import random
 import datajoint as dj
 import itertools
+import hashlib
+import uuid
+import faker
```

Comment:
It is already deterministic. The code generates deterministic UUIDs rather than random.

Reply:
@dimitri-yatsenko That is not what I meant. UUID's are generated from url which is actually generated by faker's `profile` method. [This](https://github.com/dimitri-yatsenko/datajoint-python/blob/24ba6793eafd41030e2cd9fe727a49ba105fc19c/tests/schema_simple.py#L197) is the portion I meant by not being deterministic. Each time we call `fake.profile()`, it will yield a random result unless we seed it properly.

Reply:
@dimitri-yatsenko Could you address the above comment?